src/clustering/administration/main/command_line.cc

```diff
@@ -891,7 +891,7 @@ options::help_section_t get_server_options(std::vector<options::option_t> *optio
                                              options::OPTIONAL));
     help.add("-n [ --server-name ] arg",
              "the name for this server (as will appear in the metadata).  If not"
-             " specified, it will be randomly chosen from a short list of names.");
+             " specified, one will be generated from the hostname and a random alphanumeric string.");
```

Comment:
Sounds good.
One minor thing: Could you break this string up such that the line is at most 89 characters long?